src/components/cc-tile-metrics/cc-tile-metrics.js

```diff
@@ -0,0 +1,369 @@
+    this.metricsLink = '';
+
+    /** @type {TileMetricsState} - State of the component. */
+    this.metrics = { state: 'loading' };
+
+    /** @type {boolean} */
+    this._docs = false;
+  }
+
+  _createChart (chartElement) {
+    return new Chart(chartElement, {
+      type: 'bar',
+      options: {
+        maintainAspectRatio: false,
+        radius: 0,
+        interaction: {
+          intersect: false,
+        },
+        scales: {
+          x: {
+            display: false,
+            stacked: true,
+          },
+          y: {
+            display: false,
+            beginAtZero: true,
+          },
+        },
+        plugins: {
+          legend: {
+            display: false,
+          },
+          tooltip: {
+            enabled: false,
+          },
+        },
+      },
+    });
+  }
+
+  _getChartData (inputData) {
+
+    const data = inputData ?? SKELETON_REQUESTS;
+    const labels = data.map((item) => item.timestamp);
+    const values = data.map((item) => item.value);
+
+    const maxValues = Array
+      .from(new Array(NUMBER_OF_POINTS))
+      .map((_) => 100);
+
+    const colors = inputData == null
+      ? values.map((_) => '#bbb')
+      : values.map((percent) => this._getColorChart(percent));
+
+    return {
+      labels,
+      datasets: [
+        {
+          fill: 'origin',
+          data: values,
+          backgroundColor: colors,
+        },
+        {
+          fill: 'origin',
+          data: maxValues,
+        },
+      ],
+    };
+  }
+
+  _getColorChart (percent) {
+    if (percent > TOP_THRESHOLD) {
+      return TOP_COLOR_CHART;
+    }
+    else if (percent > BOTTOM_THRESHOLD) {
+      return MIDDLE_COLOR_CHART;
+    }
+    return BOTTOM_COLOR_CHART;
+  }
+
+  _getColorLegend (percent) {
+    if (percent > TOP_THRESHOLD) {
+      return TOP_COLOR_LEGEND;
+    }
+    else if (percent > BOTTOM_THRESHOLD) {
+      return MIDDLE_COLOR_LEGEND;
+    }
+    return BOTTOM_COLOR_LEGEND;
+  }
+
+  _getCurrentPanel () {
+    if (this._docs) {
+      return 'docs';
+    }
+    if (this.metrics.state === 'error') {
+      return 'error';
+    }
+    if (this.metrics.state === 'empty') {
+      return 'empty';
+    }
+    if (this.metrics.state === 'loading' || this.metrics.state === 'loaded') {
+      return 'chart';
+    }
+  }
+
+  _onToggleDocs () {
+    this._docs = !this._docs;
+  }
+
+  firstUpdated () {
+    this._cpuCtx = this.renderRoot.getElementById('cpu_chart');
+    this._memCtx = this.renderRoot.getElementById('mem_chart');
+
+    this._cpuChart = this._createChart(this._cpuCtx);
+    this._memChart = this._createChart(this._memCtx);
+
+  }
+
+  // updated and not update because we need this._chart before
+  updated (changedProperties) {
+
+    if (changedProperties.has('metrics')) {
+
+      this._cpuChart.data = (this.metrics.state === 'loaded')
+        ? this._getChartData(this.metrics.value.cpuData)
+        : this._getChartData(null);
```

Comment:
Not sure why but I would prefer `SKELETON_REQUESTS` to be passed to `_getChartData` instead of being implicit with null.

Same for the other graph.